<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TKMessageBox</class>
 <widget class="QWidget" name="TKMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TKMessageBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>隐藏选中的条目</string>
       </property>
       <property name="text">
        <string>隐藏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="toolTip">
        <string>显示所有的条目</string>
       </property>
       <property name="text">
        <string>显示所有</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="toolTip">
        <string>显示所有的条目</string>
       </property>
       <property name="text">
        <string>变成灰色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="toolTip">
        <string>显示所有的条目</string>
       </property>
       <property name="text">
        <string>变成彩色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="toolTip">
        <string>显示所有的条目</string>
       </property>
       <property name="text">
        <string>小头像</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CMyTreeView" name="treeView">
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyTreeView</class>
   <extends>QTreeView</extends>
   <header>cmytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
